ra-tls/ra-tls-server/Makefile

```diff
@@ -12,22 +12,17 @@ SGX_SDK ?= /opt/intel/sgxsdk
 SGX_RA_TLS := $(TOPDIR)/sgx-ra-tls
 
 ifdef ECDSA
-  SGX_DCAP_URI := https://github.com/intel/SGXDataCenterAttestationPrimitives
-  SGX_DCAP_COMMIT := bfab1376480f760757738092399d0d99b22f4dfd
   SGX_DCAP ?= SGXDataCenterAttestationPrimitives
-  SGX_DCAP_INC := -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/common/inc -I$(SGX_DCAP)/QuoteGeneration/pce_wrapper/inc -I$(SGX_DCAP)/QuoteVerification/Src/AttestationLibrary/include
+  SGX_DCAP_INC := -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/common/inc -I$(SGX_DCAP)/QuoteGeneration/pce_wrapper/inc -I$(SGX_DCAP)/QuoteVerification/Src/AttestationLibrary/include -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/ql/inc -I../build/include/ -I/usr/x86_64-linux-gnu/include/ -I$(SGX_DCAP)/QuoteVerification/QVL/Src/AttestationLibrary/include/ -I/usr/include/x86_64-linux-gnu/
```

Comment:
Plz split this single line into multi-lines. And plz remove the unnecessary trailing "/".

Reply:
ok